website/cue/reference/components/sinks/splunk_hec.cue

```diff
@@ -46,6 +46,13 @@ components: sinks: _splunk_hec: {
 				}
 			}
 		}
+		default_token: {
+			description: "Your Splunk HEC token."
```

Comment:
I think we should add a note here that it'll pull it from Splunk source if `store_hec_token` is true.

Reply:
I think we need to document `store_hec_token` on the source too. I'm not seeing it here.

Reply:
Thanks for the heads up, I'd missed that. Added both changes in f68b406 and took wording from Datadog API key docs.